packages/core/src/sync/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { CID } from 'multiformats/cid'
+import { IpfsService, TreeMetadata } from './interfaces.js'
+
+const METADATA_PATH = '2'
+
+enum PathDirection {
+  L,
+  R,
+}
+
+const getPath = (index: number, leavesCount: number): PathDirection[] => {
```

Comment:
Do you have any thoughts on how we're going to test that the path logic we have here perfectly matches the logic that the CAS uses to build its merkle tree?

Reply:
yes very valid concern. I have manually tested this using real merkle root cids retrieved from the QA cas logs. I just felt for a unit test I shouldn't be relying on data that was stored via external sources. 

My plan is to have a larger test for the sync service where I use a CAS to anchor some MIDs to a test ethereum node that the syncer is hooked up to.

Reply:
Sounds good.  I also imagine we could probably import this function into the CAS codebase, which already depends on js-ceramic, and extend our unit tests that already build merkle trees of various structures with different numbers of nodes to also test this function against them.